charts/gitops-runtime/templates/gitops-operator.yaml

```diff
@@ -76,6 +76,10 @@
 
   {{- include "gitops-operator.resources" $gitopsOperatorContext}}
 
+  {{/* Set DEFAULT_TASK_PULLING_INTERVAL */}}
+  {{- if not (index .Values "gitops-operator").env.TASK_PULLING_INTERVAL }}
+    {{- $_ := set $gitopsOperatorContext.Values.env "TASK_PULLING_INTERVAL" "10s" }}
+  {{- end }}
```

Comment:
do we need this template bit?
wouldn't just adding it to the values.yaml under `gitops-operator.env` be enough to set it as a default value?